submodules/usersCallForwarding/usersCallForwarding.js

```diff
@@ -0,0 +1,551 @@
+define(function (require) {
+    var _ = require('lodash');
+    var monster = require('monster');
+
+    return {
+        usersCallForwardingRender: function (user) {
+            var self = this,
+                days = self.weekdays,
+                meta = self.appFlags.strategyHours.intervals,
+                timepickerStep = meta.timepicker.step,
+                intervalLowerBound = meta.min,
+                intervalUpperBound = meta.max,
+                intervals = [
+                    {
+                        weekday: "monday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "tuesday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "wednesday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "thursday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "friday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "saturday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                    {
+                        weekday: "sunday",
+                        start: 0,
+                        end: 84600,
+                        active: true
+                    },
+                ],
+                getData = _.bind(self.usersCallForwardingGetData, self),
+                formatData = _.bind(self.usersCallForwardingFormatData, self),
+                bindEvents = _.bind(self.usersCallForwardingBindingEvents, self),
+                initTemplate = function (data) {
+                    layoutTemplate = $(self.getTemplate({
+                        name: 'layout',
+                        data: formatData(data),
+                        submodule: 'usersCallForwarding'
+                    }));
+
+                    layoutTemplate.find('.feature-popup-title').each(function () {
+                        var strategy = $(this).data('template');
+
+                        if (strategy != 'off' || strategy != 'selective') {
```

Comment:
Instead i would do this:

```suggestion
                        if (strategy !== 'off' || strategy !== 'selective') {
```